tests/test_etter_filter.py

```
@@ -0,0 +1,239 @@
+            with pytest.raises(ef.EtterLowConfidenceError, match="Upstream rejected"):
+                p.parse("ambiguous query")
+
+    def test_low_confidence_non_strict_returns_with_warning(self):
+        upstream = MagicMock()
+        upstream.parse.return_value = SimpleNamespace(
+            spatial_relation="near",
+            reference_location="X",
+            buffer_config=None,
+            overall_confidence=0.4,
+        )
+        with patch.object(ef, "ETTER_AVAILABLE", True), \
```

Ruff (0.15.12)

[warning] Boolean positional value in function call

(FBT003)